bin/ci-rspec

```diff
@@ -50,6 +50,10 @@ if [ ! -f ${REPORTER_BIN} ]; then
   chmod 755 ${REPORTER_BIN}
 fi
 
+# update chromedriver
+bundle exec rails webdrivers:chromedriver:update
```

Comment:
Is this something that might prevent tests from sometimes failing due to chromedriver needing an update?

Reply:
webdrivers will do this for free, but VCR freaks out because it's not allowed to connect to the host, so I download them here first